api/types/kustomization_test.go

```diff
@@ -16,6 +16,67 @@ func fixKustomizationPostUnmarshallingCheck(k, e *Kustomization) bool {
 		k.Bases == nil
 }
 
+func TestKustomization_CheckDeprecatedFields(t *testing.T) {
+	tests := []struct {
+		name string
+		k    Kustomization
+		want *[]string
+	}{
+		// TODO: Add test cases.
```

Comment:
is this TODO still needed?

Reply:
A few tests are already added, and It isn't needed.
I'll delete this comment.